src/Extension.php

```diff
@@ -98,6 +110,9 @@ public function configure(ArrayNodeDefinition $builder)
                         scalarNode('dbprefix')->
                             defaultValue(null)->
                         end()->
+                        arrayNode('connection_options')->
+                            ignoreExtraKeys(false)->
```

Comment:
I think we'll need to provide a defaultValue for this node otherwise it would become a requirement for everyone to specify this node. The default value is an empty array for the PDO options.